packages/react-reconciler/src/SchedulerWithReactIntegration.new.js

```diff
@@ -51,80 +50,9 @@ if (enableSchedulerTracing) {
 
 export type SchedulerCallback = (isSync: boolean) => SchedulerCallback | null;
 
-type SchedulerCallbackOptions = {timeout?: number, ...};
-
-// Except for NoPriority, these correspond to Scheduler priorities. We use
-// ascending numbers so we can compare them like numbers. They start at 90 to
-// avoid clashing with Scheduler's priorities.
-export const ImmediatePriority: ReactPriorityLevel = 99;
-export const UserBlockingPriority: ReactPriorityLevel = 98;
-export const NormalPriority: ReactPriorityLevel = 97;
-export const LowPriority: ReactPriorityLevel = 96;
-export const IdlePriority: ReactPriorityLevel = 95;
-// NoPriority is the absence of priority. Also React-only.
-export const NoPriority: ReactPriorityLevel = 90;
-
-export const shouldYield = Scheduler_shouldYield;
-export const requestPaint =
```

Comment:
This isn't relevant anymore because Scheduler is a hard dependency of React, now and for the foreseeable future.